Core/DailyPixel.swift

```diff
@@ -46,20 +46,28 @@ public final class DailyPixel {
     private static let storage: UserDefaults = UserDefaults(suiteName: Constant.dailyPixelStorageIdentifier)!
 
     /// Sends a given Pixel once per day.
-    /// This means a pixel will get sent twice the first time it is called per-day.
```

Comment:
Removed this as it doesn't make sense (I think it was a bad copy / paste from `fireDailyAndCount` comment